config/buildout-recipe/packages.txt

```diff
@@ -9,7 +9,6 @@ z3c.recipe.i18n
 zc.recipe.cmmi
 zdaemon
 zc.recipe.filestorage
-five.customerize
```

Comment:
Was this intentional? Just wondering.

Reply:
It ended up in this PR by accident, but the change is correct, see https://github.com/zopefoundation/meta/pull/208/commits/f33185d73bbb353c449fc7ed8d0e3499fb0a00d7 – I am moving it to `master` to clean up the PR.